### PR TITLE
add job branding API documentation

### PR DESCRIPTION
**Pull request by fgrutsch**

### CODE REVIEW

File: source/includes/_jobbrandings.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
+# Job Brandings
+
+Job brandings allow you do customize your jobs to make them stand ou on social networks.
```
Comment (dpfeiffer) [inline source/includes/_jobbrandings.md]: typo

Comment (dpfeiffer) [reply source/includes/_jobbrandings.md]: also stay consistent with upper and lower case

File: source/includes/_jobbrandings.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+      "href": "https://localhost/job-header-image.jpeg"
+    },
+    "shareDescription": "This is my share description.",
+    "createdAt": "2018-10-09T10:03:39Z",
+    "updatedAt": "2018-10-09T10:04:38Z"
+  }
+]
+```
+
+`GET /v1/companies/{companyId}/job_brandings/{jobBrandingId}`
```
Comment (dpfeiffer) [inline source/includes/_jobbrandings.md]: this si the endpoint for the single job branding

File: source/includes/_jobbrandings.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+> This is an example request body
+
+```json
+{
+  "name": "Branding One",
+  "jobHeader": {
+    "id": "00000000-0000-0000-0000-000000000003"
+  },
+  "shareLogo": {
+    "id": "00000000-0000-0000-0000-000000000002"
```
Comment (dpfeiffer) [inline source/includes/_jobbrandings.md:132]: Is there already documentation on how to upload files?

**Review by dpfeiffer — CHANGES_REQUESTED** [review]
@fg-devs reviewed

File: source/includes/_jobbrandings.md (hunk 3 above)
Comment (fgrutsch) [reply source/includes/_jobbrandings.md:132]: actually not...i will add that as well

**Comment by fgrutsch** [comment]
@dpfeiffer updated

**Review by dpfeiffer — APPROVED** [review]

**Pull request merged**